crates/router/tests/connectors/sample_auth.toml

```diff
@@ -232,4 +232,6 @@ api_key="API Key"
 api_key="API Key"
 
 [adyenplatform]
+
```

Comment:
api_key is migging for adyenplatform here

Reply:
Fixed that